cmake/ycm-0.2/find-modules/FindZFP.cmake

```diff
@@ -0,0 +1,37 @@
+#
+# Find the ZFP includes and library
+#
+
+# This module defines
+# ZFP_INCLUDE_DIR, where to find zfp.h
+# ZFP_LIBRARIES, the libraries to link against
+# ZFP_FOUND, if false, you cannot build anything that requires ZFP
+
+######################################################################## 
```

Comment:
For the module documentation (and copyright attribution), you can uniform this module with the already existing ones, see for example : https://github.com/Nicogene/yarp/blob/47d4873927ae71f01eb98e3c69f3c465a3da8b83/cmake/ycm-0.2/find-modules/FindFuse.cmake .